fvm/executionParameters.go

```diff
@@ -0,0 +1,137 @@
+package fvm
+
+import (
+	"github.com/onflow/cadence"
+	"github.com/onflow/cadence/runtime"
+	"github.com/onflow/cadence/runtime/common"
+
+	"github.com/onflow/flow-go/fvm/blueprints"
+	"github.com/onflow/flow-go/fvm/errors"
+	"github.com/onflow/flow-go/fvm/meter/weighted"
+	"github.com/onflow/flow-go/fvm/utils"
+)
+
+// getExecutionWeights reads stored execution effort weights from the service account
+func getExecutionEffortWeights(
```

Comment:
Do we have a test for this function? It's rather crucial so we should have it covered.

Reply:
yes we do https://github.com/onflow/flow-go/blob/389fb3cc0adbb7727024a2e00e337500c1994894/fvm/fvm_test.go#L3581

Reply:
Does it test reading values? It seems that it check if values are respected, but given in context params

Reply:
Does it test reading values? It seems that it check if values are respected, but given in context params

Reply:
We agreed @janezpodhostnik  will add this test in new PR to unblock merge of this for the spork